db-service/lib/cqn2sql.js

```diff
@@ -290,7 +290,7 @@ class CQN2SQLRenderer {
       return `extensions__->${this.string('$."' + x.element.name + '"')} as ${x.as || x.element.name}`
     }
     ///////////////////////////////////////////////////////////////////////////////////////
-    let sql = this.expr(x)
+    let sql = this.expr({ param: false, __proto__: x })
```

Comment:
Why do we need that?

Reply:
When selecting static values it is required to always inline them. Otherwise typed databases are confused what type it is supposed to return and causes trouble with `SELECT 1` as `SELECT ?` will only accept strings as parameters as the default type of `?` is `nvarchar`. Breaking a lot of queries.